gulpfile.js

```diff
@@ -29,6 +38,68 @@ gulp.task('templates', function() {
     .pipe(gulp.dest(outputPath + 'js'));
 });
 
+/*********************************************************
+ * JavaScript
+ ********************************************************/
+
+gulp.task('js:build', function() {
+  return browserify(projectPath + 'app.js', {
+    debug: false
+  })
+    .transform(babelify, {
```

Comment:
Can we use https://github.com/moxie-leean/ng-web-app/blob/master/.babelrc for config.